numpy/ma/tests/test_extras.py

```diff
@@ -788,8 +790,7 @@ def test_1d_w_missing(self):
         nx = x.compressed()
         assert_almost_equal(np.cov(nx), cov(x))
         assert_almost_equal(np.cov(nx, rowvar=False), cov(x, rowvar=False))
-        assert_almost_equal(np.cov(nx, rowvar=False, bias=True),
```

Comment:
Weird, have to try it. Likely an error because of mistaking it with corrcoef where it wouldn't test much at all since the bias kwarg is deprecated.